app/modules/account.js

```diff
@@ -18,8 +19,15 @@ export function logout(){
   }
 };
 
+export function showKey(isToggled) {
```

Comment:
Thanks, I agree it's not an app-wide state, but I'm not entirely sure how to contain this state change to `Login` only, could you elaborate further? (New to redux but not react, apologies)